src/stream/src/task/stream_manager.rs

```diff
@@ -416,15 +417,15 @@ impl LocalStreamManagerCore {
                         .iter()
                         .map(|i| *i as usize)
                         .collect();
-                    let hash_mapping = dispatcher
-                        .hash_mapping
-                        .clone()
-                        .ok_or_else(|| {
-                            RwError::from(ErrorCode::InternalError(
-                                "hash dispatcher doesn't have consistent hash mapping".to_string(),
-                            ))
-                        })?
-                        .hash_mapping;
+                    let compressed_mapping = dispatcher.hash_mapping.clone().ok_or_else(|| {
```

Comment:
We can get the reference with `get_xx`.
```suggestion
                    let compressed_mapping = dispatcher.get_hash_mapping()?;
```